ml-agents/mlagents/trainers/settings.py

```diff
@@ -199,28 +199,37 @@ def structure(d: Mapping, t: type) -> Any:
             enum_key = RewardSignalType(key)
             t = enum_key.to_settings()
             d_final[enum_key] = strict_to_cls(val, t)
+            if "encoding_size" in val:
```

Comment:
Backward compatible with old configs

Reply:
Can you ad a comment around this code so we will remember it ?